app/src/main/res/layout-sw600dp-land/question_player_fragment.xml

```diff
@@ -95,7 +95,34 @@
         android:progressDrawable="@drawable/progress_bar"
         app:layout_constraintBottom_toTopOf="@id/question_progress_text"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent" />
+        app:layout_constraintStart_toEndOf="@id/hints_and_solution_fragment_container" />
```

Comment:
Update this to `app:layout_constraintStart_toStartOf="parent" />`
 because marginStart=96 and marginEnd=96dp makes sure that the progressbar never overlaps with the `hints_and_solution_fragment_container `

So technically we don't need any change in this file except that of `android:layout_width="0dp"` in ProgressBar

Reply:
okk.done

Reply:
This is still not correct.
Remove `app:layout_constraintStart_toEndOf="@id/hints_and_solution_fragment_container"`
and add `app:layout_constraintStart_toStartOf="parent"` and make ProgressBar width `android:layout_width="0dp"`